### ISSUE TITLE
Excuse me, can you train with a single GPU? I only have one GPU available at the moment!

### ISSUE THREAD

**Issue by YeJiaJun321** [charlesCXK/TorchSemiSeg#5]

**Comment by charlesCXK** [comment]
Hi, one GPU is able to run the experiments. However, due to the batch size in semantic segmentation is important, I am not sure the performance will be reproduced with only one GPU.

To run the experiments, you need to:
```
1. Change the batch size and the learning rate accordingly.
2. Change the image crop size during training and inference. e.g., C.image_height in config.py
```

**Comment by woaicv** [comment]
Sorry to bother you, I'm a little white in research. I want to reproduce your code. At present, I have a GPU model of 3060, and the batchsize can only be adjusted to 2. Then I set the learning rate to 0.005. Would you like to ask whether epoch should be adjusted accordingly. I haven't changed the crop size yet. Do I need to change it?

**Comment by charlesCXK** [comment]
@woaicv Hi, the learning rate should be proportional to the batch size.
